src/util/analytics-client.ts

```diff
@@ -34,6 +37,50 @@ const calculateClientEnv = () => {
 	return "dev";
 };
 
+interface SQSEventPayload {
+	accountId?: string;
+	eventType: "screen" | "ui" | "operational" | "track";
+	eventProps: ScreenEventProps | TrackOpUiEventProps;
+	eventAttributes?: Record<string, unknown>;
+}
+
+export interface SQSAnalyticsMessagePayload {
+	jiraHost: string;
+	eventPayload: SQSEventPayload;
+}
+
+export const sendAnalyticsWithSqs = async (
+	jiraHost: string,
+	eventType: "screen" | "ui" | "operational" | "track",
+	eventProps: (ScreenEventProps | TrackOpUiEventProps)  & Record<string, unknown>,
+	attributes: Record<string, unknown> = {},
+	accountId?: string
+): Promise<void> => {
+
+	const payload: SQSAnalyticsMessagePayload = {
+		jiraHost,
+		eventPayload: {
+			accountId,
+			eventType,
+			eventProps,
+			eventAttributes: attributes
+		}
+	};
+	const sqs = new SQS({ apiVersion: "2012-11-05", region: envVars.SQS_INCOMINGANALYTICEVENTS_QUEUE_REGION });
+	const params: SendMessageRequest = {
+		MessageBody: JSON.stringify(payload),
+		QueueUrl: envVars.SQS_INCOMINGANALYTICEVENTS_QUEUE_URL,
+		DelaySeconds: 0
+	};
+	try {
```

Comment:
/nit
Is there a reason the `try{` doesn't move to the very top of this method?

Reply:
While "sendMessage" is anticipated to result in a error from time to time, and we are OK to ignore it (this is analytics), if there's a bug above - that's not anticipated and we should uncover this ASAP.